extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -16,7 +16,7 @@ import { ProjectType } from 'mssql';
 
 const buildDirectory = 'BuildDirectory';
 const sdkName = 'Microsoft.Build.Sql';
-const microsoftBuildSqlDefaultVersion = '0.1.9-preview'; // default version of Microsoft.Build.Sql nuget to use for building legacy style projects
+const microsoftBuildSqlDefaultVersion = '0.1.9-preview'; // default version of Microsoft.Build.Sql nuget to use for building legacy style projects, update in README when updating this
```

Comment:
Hmm, I didn't realize this was defined here.  ...I should do a pass at this to ensure it's not going to accidentally use a different `Microsoft.Build.Sql` version from what's defined in the sqlproj...

Reply:
this is only used for determining which SDK version to use for building legacy sql projects and is configurable in the user settings. For legacy sql projects, the extension downloads the Microsoft.Build.Sql SDK nuget, then unzips it and copies the dlls to the BuildDirectory folder to use for building. Before, we used to copy those dlls from STS, but ran into an issue when the Migration nuget increased a version of one of the System dlls and that broke building sql projects.

Reply:
Gotchya.  In that case, the description text for the extension setting

> Version of Microsoft.Build.Sql to use for SDK-style SQL projects. If not set, the extension will use 0.1.9-preview.

Should be "to use for legacy/SSDT-style SQL projects" ?